conftest.py

```diff
@@ -0,0 +1,6 @@
+import pytest
+
+
+@pytest.fixture(scope="session")
+def pipeline_testdir(tmp_path_factory):
```

Comment:
is there a reason to use `tmp_path_factory` instead of `tmp_path`? Seems overkill

Reply:
`tmp_path_factory` is a session-scoped fixture, whereas `tmp_path` is a function-scoped one

the end result is that by using the simpler tmp_path each test creates its own directory and the test outputs are then stored all separately

Reply:
ok